USAGE.md

```diff
@@ -121,6 +125,76 @@ Bye Bye 😔
 
    ![ubuntu-connected](https://i.imgur.com/VJVacKe.png)
 
+### Installing in a virtual environment
+
+If you're having trouble with the normal installation or don't want to install ProtonVPN-CLI as root, follow this guide to install it in a Python virtual environment.
+
+#### Install
+
+1. Create a directory for the Installation and step into it
+
+    `mkdir ~/Documents/ProtonVPN-CLI`
+
+    `cd ~/Documents/ProtonVPN-CLI`
+
+2. Install the virtualenv Python package
+
+    `pip3 install virtualenv --user`
+
+3. Create a virtual environment and activate it
+
+    `virtualenv .venv`
+
+    `source .venv/bin/activate`
+
+4. Now that you're in the virtual environment, install ProtonVPN-CLI
+
+    `pip install protonvpn-cli`
+
+    As you're in the virtualenv, `pip` should be the same as `pip3`.
+
+5. You should now have the executable `.venv/bin/protonvpn`.
+
+    `which protonvpn`
+
+6. If that works, deactivate the virtual environment again
+
+    `deactivate`
+
+7. Link the executable from above (output of the `which` command) to a PATH folder so you can access it from anywhere
+
+    `sudo ln -sf ~/Documents/ProtonVPN-CLI/.venv/bin/protonvpn /usr/local/bin/protonvpn`
```

Comment:
Maybe better add the path to the PATH variable?

Reply:
I don't think that's right. The `.venv/bin/` directory also contains executables for interacting with the virtual environment as well as the pip and python binaries for the virtual environment.